StripeConnect/StripeConnect/Source/Internal/Webview/ConnectComponentWebView.swift

```diff
@@ -48,25 +48,22 @@ class ConnectComponentWebView: ConnectWebView {
 
         super.init(frame: .zero, configuration: config)
         componentManager.registerChild(self)
-        guard let publishableKey = componentManager.apiClient.publishableKey else {
-            assertionFailure("A publishable key is required. For more info, see https://stripe.com/docs/keys")
-            return
-        }
```

Comment:
This is now handled in `ConnectJSURLParams` when we call `STPAPIClient.validateKey`